docs/creating-and-editing-specialist-document-types.md

```diff
@@ -70,19 +65,20 @@ Finally, you'll need to add your custom fields to:
 
 ## 4. Configure the email sign up page
 
-The email sign up page is rendered by [Finder Frontend](https://github.com/alphagov/finder-frontend) using the configuration in the new schema added to specialist publisher.
+The email sign up page is rendered by [Finder Frontend](https://github.com/alphagov/finder-frontend) using the configuration in the new schema added to specialist publisher. The schema should specify `email_filter_by` and `email_filter_facets` (e.g. [cma-cases](https://github.com/alphagov/specialist-publisher/blob/ce68fdb008cab05225e0493e19decba5365e1e20/lib/documents/schemas/cma_cases.json#L29)).
 
 If your email sign up page should have checkboxes (e.g. [cma-cases](https://www.gov.uk/cma-cases/email-signup)), you will need to edit email-alert-api by adding the new tags to [valid_tags.rb](https://github.com/alphagov/email-alert-api/blob/3e0018510ea85f5d561e2865ad149832b94688a1/lib/valid_tags.rb#L2).
 
 ## 5. Deploy and publish
 
 To deploy:
 
-1. Deploy Specialist Publisher and Search API (and govuk-content-schemas if you haven't already).
+1. Deploy Publishing API, Specialist Publisher, Search API and Email Alert API.
```

Comment:
We mention deploying publishing API on line 30, so either we should change this to if you haven't already deployed publishing API do it now. And we should make it clear that publishing API must be deployed before the other apps (or else we'll get validation errors)

And maybe we should change Email Alert API to add an if necessary, since not all finders have emails so not all of them need an email PR